backend/src/tasks/functions.yml

```diff
@@ -363,3 +363,45 @@ byOrgCache:
     - schedule:
         rate: cron(0 0 * * ? *)  # This triggers the function every day at midnight
   iamRoleStatements: []
+
+ecsRemediator:
+  <<: *adot
+  handler: src/xfd_django/xfd_api/tasks/ecs_remediator.lambda_handler
+  timeout: 60
+  memorySize: 512
+  environment:
+    # 1. Use existing SSM var for Cluster (ARN is valid for 'cluster' arg)
+    ECS_CLUSTER_NAME: ${ssm:/crossfeed/${self:provider.stage}/WORKER_CLUSTER_ARN}
+
+    # 2. Use NEW SSM var for Service (Must be a name, not a Cluster ARN)
+    ECS_SERVICE_NAME: ${ssm:/crossfeed/${self:provider.stage}/worker_ecs_service_name}
```

Comment:
Can we just use the env defined in the serverless.yml like all the other functions use? Same with project and Stage. That shouldn't need to be repeated here.

The variable though (SNS_ALARMS_TOPIC_ARN) I'm fine to define here, so it's only used in this function